inc/admin/tipologie/tipologia_luogo.php

```diff
@@ -280,9 +280,9 @@ function dci_add_luogo_metaboxes() {
     ) );
 
     $cmb_dove->add_field( array(
-        'id'         => $prefix . 'quartiere',
-        'name'       => __( 'Quartiere  ', 'design_comuni_italia' ),
-        'desc'       => __( 'Se il territorio è mappato in quartieri, riportare il Quartiere dove è situato il luogo.', 'design_comuni_italia' ),
+        'id'         => $prefix . 'locality',
```

Comment:
Forse non è opportuno sostituire del tutto "quartiere". [Nell'architettura dell'informazione](https://designers.italia.it/files/resources/modelli/comuni/adotta-il-modello-di-sito-comunale/definisci-architettura-e-contenuti/Architettura-informazione-sito-Comuni.ods), nella Tipologia Luogo, "Quartiere" è indicato (anche se non obbligatorio), e "Località" no.
Però, è molto probabile che una correzione sulla struttura dei dati del Luogo sia necessaria.